### PR TITLE
Added support for light and blade types

### PR DESCRIPTION
**Pull request by Dimitri23**
This adds support for blade openers and lights.

### CODE REVIEW

File: pyvlx/lightening_device.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
+"""Module for window openers."""
```
Comment (Julius2342) [inline pyvlx/lightening_device.py]: C&P Error

File: pyvlx/parameter.py [hunk 2]
```diff
@@ -7,6 +7,7 @@ class Parameter():
 
     UNKNOWN_VALUE = 63487  # F7 FF
     CURRENT_POSITION = 53760  # D2 00
+    CURRENT_INTENSITY = 53760  # D2 00
     MAX = 51200  # C8 00
     MIN = 0  # 00 00
     ON = 0  # 00 00
```
Comment (Julius2342) [inline pyvlx/parameter.py]: I think adding a `  #pylint disable=invalid-name` should be added here. "ON" is exactly the right name.

File: pyvlx/lightening_device.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        Parameters:
+            * wait_for_completion: If set, function will return
+                after device has reached target intensity.
+
+        """
+        await self.set_intensity(
+            intensity=Intensity(intensity_percent=100),
+            wait_for_completion=wait_for_completion)
+
+    async def stop(self, wait_for_completion=True):
```
Comment (Julius2342) [inline pyvlx/lightening_device.py]: Does a stop for a dimming device make sense?

File: pyvlx/lightening_device.py (hunk 1 above)
Comment (Dimitri23) [reply pyvlx/lightening_device.py]: Corrected

File: pyvlx/lightening_device.py (hunk 3 above)
Comment (Dimitri23) [reply pyvlx/lightening_device.py]: You're right. I remove it.

File: pyvlx/parameter.py (hunk 2 above)
Comment (Dimitri23) [reply pyvlx/parameter.py]: Sorry, I do not understand.

**Comment by Dimitri23** [comment]
I am trying to add tests for lights but I have to learn how to do it before.

File: pyvlx/parameter.py (hunk 2 above)
Comment (Julius2342) [reply pyvlx/parameter.py]: [Pylint](https://www.pylint.org/) is complaining, that "ON" is not a correct name for a variable. I wanted to say, that we should ignore this warning.

**Comment by Dimitri23** [comment]
I just added tests for Light and Blade types.

**Comment by Julius2342** [comment]
But it looks like they are failing :-/

**Comment by Dimitri23** [comment]
After few commits, tests failed on changes that is not mine.

**Comment by Dimitri23** [comment]
Hello,
Is it possible to merge commits please?
Last failing checks are concerning code that I do not modify,
I do not know how to fix them.
Dimitri

**Pull request merged**